automatic/vcredist140/vcredist140.nuspec

```diff
@@ -3,7 +3,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>vcredist140</id>
-    <version>14.32.31332</version>
+    <version>14.32.31332.2</version>
```

Comment:
Do not update the version for automatic packages in this repository.

Please revert this change.

Reply:
Reverted.